src/Assets/TestProjects/PackageValidationTestProject/Program.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace PackageValidationTestProject
+{
+    public class Program
```

Comment:
Should we also add #ifdefs in here so that we can test for non-success validation errors? Or is that not intended as part of these tests? I was thinking that if you wanted to fail the tests we could add here something very simple like:
```c#
public class Program
{
#if ForceValidationProblem
#if !NET6_0
  public void SomeAPINotIn6_0()
  {
  }
#endif
#endif
```
That way when you want to cause a validation error you just pass in a /p:ForceValidationProblem=true which you should have in the csproj a defineconstant then so that the APICompat issue is triggered.

Reply:
The purpose of adding a test that fails validation would be to ensure that a failed validation would fail the msbuild.exe invocation. I know that validation itself has it's own tests so it's ok not to have an exhaustive testing in here, but at least also ensure that we catch regressions where validation failure doesn't trigger msbuild failure would be good.